server/area_manager.py

```diff
@@ -462,7 +462,7 @@ class Testimony:
             
             def __init__(self, title):
                 self.title = title
-                self.statements = []
+                self.statements = [] * 30 # if you have more than 30 statements, don't.
```

Comment:
This is not the correct way to make a static array in Python. Your `* 30` is a no-op. Use bounds checking instead.

Reply:
aw crud